DigitalLearningSolutions.Web/ViewModels/LearningMenu/PostLearningAssessmentViewModel.cs

```diff
@@ -30,11 +32,15 @@ public PostLearningAssessmentViewModel(PostLearningAssessment postLearningAssess
             if (postLearningAssessment.PostLearningAttempts == 0)
             {
                 AssessmentStatus = "Not attempted";
+                StartButtonText = "Start assessment";
+                StartButtonAdditionalStyling = "";

```

Comment:
Isn't this the default value? 🙂

Reply:
I'm not quite sure, what you mean, maybe the empty string on line 36? I think so but it was giving a "value might not be assigned" warning without me explicitly setting it to 0 here 🙂

Reply:
Ah in which case assigning it to empty string when defining the variable would probably make the error go away. But since you changed it to use a ternary operator it's fine as it is 🙂